test/test_transforms.py

```diff
@@ -2136,6 +2139,121 @@ def test_transform_no_env(self, device, batch):
         pytest.skip("TrajCounter cannot be called without env")
 
 
+class TestStack(TransformBase):
```

Comment:
when testing a transform, we need to check that they behave correctly in single env, multi-env, replay buffers and modules. You can go through the abstract methods in the base class and check other tests (eg CatTensor) to see an example